pkg/openstackbaremetalset/baremetalhost.go

```diff
@@ -70,7 +70,11 @@ func BaremetalHostProvision(
 	// User data cloud-init secret
 	if userDataSecret == nil {
 		templateParameters := make(map[string]interface{})
-		templateParameters["AuthorizedKeys"] = strings.TrimSuffix(string(sshSecret.Data["authorized_keys"]), "\n")
+		// Prepare ssh_authorized_keys list for template
+		splitKeys := strings.Split(strings.TrimSuffix(string(sshSecret.Data["authorized_keys"]), "\n"), "\n")
+		sshKeys := make([]string, len(splitKeys))
```

Comment:
You don't need to do this as in @abays' patch.. strings.Split() returns a slice.

Reply:
I've shamelessly copy-pasted code then - with mention of the original PR in the comment. cc @abays - guess we should be good?